src/pages/connect.vue

```diff
@@ -37,7 +37,7 @@ export default {
         this.$store.dispatch("checkout/setTransactionData", this.$store.getters["checkout/getTransactionData"]);
       } else {
         this.$store.dispatch("checkout/requestInitialData");
-        await this.$router.push("/");
+        await this.$router.push({query: this.$route.query, path: "/"});
       }
     },
     async walletConnect() {
```

Comment:
Your methods are copy-paste. It should be refactored:
a) either explode repeated code into third method leaving existing too;
b) or implode it into a single method with the incoming param to choose right auth method.

First way is better but both are valid